geonode/local_settings.py.geoserver.sample

```diff
@@ -179,119 +248,6 @@ LAYER_PREVIEW_LIBRARY = 'geoext'
 #    'RESET_VIEW': False
 #}
 
-GEONODE_APPS = (
```

Comment:
Are those supposed to be removed from here?

Reply:
Yes. No need to rewrite the ones from "settings.py"